code/game/objects/structures/droppod.dm

```diff
@@ -363,6 +363,8 @@ GLOBAL_LIST_INIT(blocked_droppod_tiles, typecacheof(list(/turf/open/space/transi
 /obj/structure/drop_pod_launcher/attack_powerloader(mob/living/user, obj/item/powerloader_clamp/attached_clamp)
 	if(!istype(attached_clamp.loaded, /obj/structure/droppod))
 		return ..()
+	for(var/atom/movable/ejectee AS in contents) // dump them out, just in case no mobs get deleted
+		ejectee.forceMove(loc)
```

Comment:
Isn't this dumping them out when the pod gets put down, not when it's picked up?

Reply:
Wait yeah I put it in the wrong place